Universal-Federated-Analytics.js

```diff
@@ -1234,7 +1299,12 @@ Digital Analytics Program Government Wide Site Usage Measurement and Tracking.
 
         if ((_param[0].match(new RegExp(checkParams)) != null && _param[0].match(new RegExp(UncheckParams)) != null) || /query|json|default/ig.test(type)) {
           piiRegex.forEach(function (pii) {
-            _val = _val.replace(pii.regex, '[REDACTED_' + pii.name + ']');
+            if ((/^lat$/i.test(_param[0]) && /^(-?[1-8]?\d(?:\.\d{1,18})?|90(?:\.0{1,18})?)$/.test(_val)) || (/^lon$/i.test(_param[0]) && /^(-?(?:1[0-7]|[1-9])?\d(?:\.\d{1,18})?|180(?:\.0{1,18})?)$/.test(_val))) {
+              //if(!/^(lon|lat)$/i.test(_param[0]) || (/^(lon|lat)$/i.test(_param[0]) && (_val && !isFinite(_val)) ) ){
```

Comment:
This is an empty block.  Is there supposed to be some logic that happens when the `if` condition is true?  Why is the commented code here?

Reply:
If this is intended to skip redaction for the case when the condition is true, then you could have a `return` in the `if` block.

Reply:
The intent is to skip the redaction (do nothing) if the query parameter is a valid longitude or latitude, otherwise, these will be redacted using the same logic as other parameters.